### ISSUE TITLE
Build fail with php-8.0 - patch

### ISSUE THREAD

**Issue by barjac** [sadko4u/lsp-plugins#167]
Build fails with php-8.0 due to obsolete syntax in  lsp-plugins-1.1.28/src/doc/inc/plugins.php (line 8).

Seems fixed with:
```
diff -ur lsp-plugins-1.1.28/src/doc/inc/plugins.php lsp-plugins-1.1.28n/src/doc/inc/plugins.php
--- lsp-plugins-1.1.28/src/doc/inc/plugins.php	2020-12-20 22:10:45.000000000 +0000
+++ lsp-plugins-1.1.28n/src/doc/inc/plugins.php	2020-12-21 14:37:16.732659027 +0000
@@ -5,7 +5,8 @@
 	{
 		$a = $a['description'];
 		$b = $b['description'];
-		return ($a == $b) ? 0 : ($a > $b) ? 1 : -1;
+		 return ($a == $b) ? 0 : (($a > $b) ? 1 : -1);
+
 	}
 	
 	function sort_plugins()


```

**Comment by sadko4u** [comment]
Thanks. Applied the patch: d002af89c47650d9ed9f9b0b3e64d8d52e061800

**Comment by sadko4u** [comment]
Added patch to the **hotfix** branch: 1d9a44dfe659dee751d6f72ebaec07a1234ccde0

**Comment by sadko4u** [comment]
Available in 1.1.29!
